ci/do_coverity_local.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+#  do_coverity_local.sh
+#
+#  This script builds Envoy with the Coverity Scan Built Tool.
+#
+#  It expects the following environment variables to be set:
+#    COVERITY_TOKEN      - set to the user's Coverity Scan project token.
+#    COVERITY_USER_EMAIL - set to the email address used with the Coverity account.
+#                          defaults to the local git config user.email.
+
+
+set -e
+
+. ./ci/envoy_build_sha.sh
+
+[[ -z "${ENVOY_DOCKER_BUILD_DIR}" ]] && ENVOY_DOCKER_BUILD_DIR=/tmp/envoy-docker-build
+mkdir -p ${ENVOY_DOCKER_BUILD_DIR}
+
+TEST_TYPE="bazel.coverity"
+COVERITY_USER_EMAIL=${COVERITY_USER_EMAIL:-$(git config user.email)}
```

Comment:
Quote here as well, even though there should be no spaces in e-mail, it's just a good shell style idiom.